### DISCUSSION TITLE
PS acv

### DISCUSSION THREAD

**Discussion by Yakonrus** [ImageMagick/ImageMagick#3823] [Help]
Hello.
Is it possible to use color correction from Photoshop?
Curves preset .acv
![изображение](https://user-images.githubusercontent.com/4978662/121813808-65240a00-cc76-11eb-9c40-0ec2dc16ed57.png)

**Comment by fmw42** [comment]
Can you export the preset file so we can see what is in it an its format?  If so, perhaps someone can provide a script to read and use it. But please identify your ImageMagick version and platform/OS.

**Reply by Yakonrus** [reply]
Windows 10, .NET
I am using Magick.NET 7.24.1
https://github.com/dlemstra/Magick.NET 
 [Untitled.zip](https://github.com/ImageMagick/ImageMagick/files/6644244/Untitled.zip)
In ffmpeg it is easy to use "curves=psfile='MyCurvesPresets/purple.acv"

**Comment by fmw42** [comment]
The file is not a text file that can be read by a text editor. It is binary and I do not have any way to read it.

**Reply by Yakonrus** [reply]
https://github.com/rocketlobster/curver
I thought maybe during this time there was built-in support.

**Reply by Yakonrus** [reply]
https://github.com/MatthewRuddy/jQuery-filter.me/blob/master/jquery.filterme.js

**Comment by fmw42** [comment]
Not that I know about. But the IM developers would have to comment further.

**Reply by fmw42** [reply]
One way to transfer the curves to ImageMagick would be to use HALD clut.  Create a HALD image and save it as PNG on ImageMagick. Open it in Photoshop and apply your curve to it. Then save that file again as PNG (or any lossless compressed format). Bring that file back to ImageMagick and apply it using -hald-clut.  See https://legacy.imagemagick.org/Usage/color_mods/#hald-clut

**Reply by Yakonrus** [reply]
Unfortunately, then it doesn't make any sense.
It will be difficult for users.
The essence is as follows.
The photographer adjusts the color correction in Photoshop, saves,
and then the images are automatically processed in my software using this color correction.
I am looking for a way to directly apply this file to my software.
